scripts/main.js

```diff
@@ -0,0 +1,57 @@
+let books;
+// eslint-disable-next-line
```

Comment:
While you are allowed to disable ESLint rules for this particular milestone, you should only do so if it relates to having multiple classes in the same file or having unused variables. None of your rules disabling match these specifications. Please do not disable ESLint rules unless they are related to the scenarios described above.

<img width="1069" alt="Screen Shot 2022-08-02 at 7 51 09" src="https://user-images.githubusercontent.com/82426682/182378961-689531cd-6eb1-4396-85d8-b63681e251b1.png">